app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```
@@ -314,7 +221,11 @@ $search-icon-size: 20px;
 }
 
 .gem-c-layout-super-navigation-header__navigation-items {
-  .js-module-initialised & {
+  @include govuk-media-query($from: "desktop") {
+    display: inline-block;
+  }
+
+  .js-enabled & {
```

We'd want to swap this out for `.js-module-initialised` in line with https://github.com/alphagov/govuk_publishing_components/pull/2388

```suggestion
  .js-module-initialised & {
```

Same thing on lines 370, 538 and 859.

---

Good catch; swapped the classes back in https://github.com/alphagov/govuk_publishing_components/pull/2401/commits/f1893d5d62368fa84a02ae0b64d3d9a6a76b556c.